hudi-common/src/main/java/org/apache/hudi/common/config/HoodieCommonConfig.java

```diff
@@ -72,6 +72,14 @@ public class HoodieCommonConfig extends HoodieConfig {
       .markAdvanced()
       .withDocumentation("Turn on compression for BITCASK disk map used by the External Spillable Map");
 
+  public static final ConfigProperty<Boolean> INCREMENTAL_FETCH_INSTANT_BY_STATE_TRANSITION_TIME = ConfigProperty
+      .key("hoodie.incremental.fetch.instant.by.state.transition.time")
+      .defaultValue(false)
```

Comment:
`hoodie.incremental.fetch.instant.by.state.transition.time` seem not a good name, does
`hoodie.datasource.read.by.state.transition.time` make sense to you?